File: Android/app/src/main/java/com/nfsindustries/cryptocurrencymonitor/CryptoCurrencyMonitor.java
package com.nfsindustries.cryptocurrencymonitor;

import android.app.Application;
import android.content.Context;

/**
 * Created by maxmo on 29/5/2017.
 */

public class CryptoCurrencyMonitor extends Application {

    private static Context context;

    public void onCreate() {
        super.onCreate();
        CryptoCurrencyMonitor.context = getApplicationContext();
    }

    public static Context getAppContext() {
        return CryptoCurrencyMonitor.context;
    }
}

File: Android-Kotlin/app/src/main/java/com/nfsindustries/cryptocurrencymonitor/CryptoCurrencyMonitor.kt
package com.nfsindustries.cryptocurrencymonitor

import android.app.Application
import android.content.Context

/**
 * Class used for Context and support functions
 * Created by MaxMouse on 21/9/2017.
 */
class CryptoCurrencyMonitor : Application() {

    override fun onCreate() {
        super.onCreate()
        appContext = applicationContext
    }

    companion object {

        var appContext: Context? = null
            private set
    }
}
